src/app/components/modal-window/modal-window.service.ts

```diff
@@ -5,15 +5,15 @@ import { Subject } from 'rxjs';
   providedIn: 'root',
 })
 export class ModalWindowService {
-  public visible = new Subject<boolean>();
+  public visible = new Subject<any>();
```

Comment:
Looks like bad change, why you made it?

Reply:
because unsubcribe isn't want work with boolean... 
![image](https://user-images.githubusercontent.com/78950448/141432905-b5e602d9-1b82-43dd-8ac7-6ccfeb22c77a.png)